### PR TITLE
fix(learn): enhance user stories for header parser project

### PR DESCRIPTION
**Pull request by moT01**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Related to: #39721

### CODE REVIEW

**Review by ShaunSHamilton — DISMISSED** [review]
LGTM 👍

File: curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/request-header-parser-microservice.md [hunk 1]
```diff
@@ -7,9 +7,13 @@ forumTopicId: 301507
...
+Build a full stack JavaScript app that is functionally similar to this: <a href='https://request-header-parser-microservice.freecodecamp.rocks/' target='_blank'>https://request-header-parser-microservice.freecodecamp.rocks/</a>. Working on this project will involve you writing your code using one of the following methods:
+
+- Clone <a href='https://github.com/freeCodeCamp/boilerplate-project-headerparser/'>this GitHub repo</a> and complete your project locally.
+- Use <a href='https://repl.it/github/freeCodeCamp/boilerplate-project-headerparser' target='_blank'>our repl.it starter project</a> to complete your project.
+- Use a site builder of your choice to complete the project. Be sure to incorporate all the files from our GitHub repo.
+
+When you are done, make sure a working demo of your project is hosted somewhere public. Then submit the URL to it in the `Solution Link` field. Optionally, also submit a link to your projects source code in the `GitHub Link` field.
```
Comment (Manish-Giri) [inline curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/request-header-parser-microservice.md:16]:
```suggestion
When you are done, make sure a working demo of your project is hosted somewhere public. Then submit its URL in the `Solution Link` field. Optionally, also submit a link to your project's source code in the `GitHub Link` field.
```

**Review by Manish-Giri — CHANGES_REQUESTED** [review]
A typo fix and a minor suggestion.

File: curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/request-header-parser-microservice.md (hunk 1 above)
Comment (moT01) [reply curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/request-header-parser-microservice.md:16]: I think what you suggested sounded slightly better, but I just added the apostrophe. Trying to stick with a universal wording for these and there's a number of PR's to make it like this already so I think we should stick with that.

**Review by RandellDawson — APPROVED** [review]
LGTM

**Comment by RandellDawson** [comment]
@Manish-Giri If you are fine with the changes, approve and merge.

Thanks

**Review by Manish-Giri — APPROVED** [review]

**Pull request merged**